README.md

```diff
@@ -29,13 +29,13 @@ If you're not sure if that's all installed or you don't know how then check out
 
 Using the standard go tools you can install Fyne's core library using:
 
-    $ go get fyne.io/fyne/v2
+    $ go install fyne.io/fyne/v2
 
 # Widget demo
 
 To run a showcase of the features of Fyne execute the following:
 
-    $ go get fyne.io/fyne/v2/cmd/fyne_demo/
+    $ go install fyne.io/fyne/v2/cmd/fyne_demo/
```

Comment:
It is probably also worth noting that this would fail on Go 1.16 and higher because it requires to add `@latest` at the end when using that command.